package.json

```diff
@@ -20,9 +20,7 @@
     "babel-core": "^5.8.21",
     "bootstrap": "^3.3.4",
     "clear-cut": "^2.0.1",
-    "coffee-cash": "0.8.0",
```

Comment:
Woah thanks for the heads up. PR for that here: https://github.com/atom/timecop/pull/13